.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Unit Tests
+on: [push, pull_request]
+jobs:
+  test:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        gemfile:
+        - Gemfile
+        - gemfiles/rails_4.gemfile
```

Comment:
Probably don't need rails 4 support anymore right?

Reply:
I'll remove this as a follow on